docs/application/tizen-studio/native-tools/ui-builder/databind.md

```diff
@@ -5,7 +5,7 @@ Databinding helps you to manage application-specific data sources and models alo
 In the **Data Binding** view, you can, for example, add data sources, create data models from those sources, and bind data elements to UI components in the **Design** tab. The view displays all the data sources that you have added to your application. Along with the data sources, you can also see the "data element – UI component" bindings that you have created.
 
 > **Note**  
-> Databinding is supported for 2.4 and higher mobile applications, and 3.0 and higher wearable applications.When creating the application, you must use the [Project Wizard](project-wizard-n.md) and select a native UI Builder template (any template with the "UI Builder" prefix).
+> Databinding is supported for 2.4 and higher mobile applications, and 3.0 and higher wearable applications.When creating the application, you must use the [Project Wizard](project-wizard.md) and select a native UI Builder template (any template with the "UI Builder" prefix).
```

Comment:
It looks `../project-wizard.md` is correct.

Reply:
done